src/glabs-left/web/index.scss

```diff
@@ -0,0 +1,95 @@
+@import '_core';
+
+a {
+  text-decoration: none;
+}
+
+h1, p {
+  margin: 0;
+}
+
+.container {
+  font-family: "Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;
+  background-color: $neutral-8;
+  border-top: 2px solid $paid-article-brand;
+
+  display: flex;
+  flex-direction: column;
+
+
+  width: $gs-gutter * 6.5;
+
+  @include mq(tablet) {
+    width: $gs-gutter * 11;
+  }
+}
+
+.margin {
+  margin: 5px;
+}
+
+.body {
+  flex-grow: 1;
+  flex-shrink: 1;
+  flex-basis: auto;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.thumbnail {
+  max-height: 40%;
+  overflow:hidden;
```

Comment:
Nit (missing space): `overflow: hidden;`

...Although I'm hoping to add sass-lint to the project 🙃